configuration/configure-automatic-upgrades.adoc

```diff
@@ -1,18 +1,21 @@
 [id="configure-automatic-upgrades"]
-= Configuring automatic upgrades for secured clusters
+= Configuring automatic upgrades for manifest-installed secured clusters
 include::modules/common-attributes.adoc[]
 :context: configure-automatic-upgrades
 
 toc::[]
 
 [role="_abstract"]
-You can automate the upgrade process for each secured cluster and view the upgrade status from the {product-title-short} portal.
+If you installed {product-title} by using the manifest installation method, also known as the `roxctl` CLI method, you can automate the upgrade process for each secured cluster. You can also view the upgrade status from the {product-title-short} portal.
 
-Automatic upgrades make it easier to stay up-to-date by automating the manual task of upgrading each secured cluster.
+[NOTE]
+====
+Automatic upgrades are only available for {product-title-short} systems that were installed by using the manifest installation method. If you installed {product-title-short} by using the Operator, upgrades are controlled by using {olm-first}. If you installed {product-title-short} by using Helm charts, you must use Helm to upgrade.
+====
 
-With automatic upgrades, after you upgrade Central;  Sensor, Collector, and Compliance services in all secured clusters, automatically upgrade to the latest version.
+Automatic upgrades make it easier to stay up-to-date by automating the manual task of upgrading each secured cluster. With automatic upgrades, after you upgrade Central, then Sensor, Collector, and Compliance services in all secured clusters automatically upgrade to the latest version.
 
-{product-title} also enables centralized management of all your secured clusters from within the {product-title-short} portal.
+{product-title-short} also enables centralized management of all your secured clusters from within the {product-title-short} portal.
```

Comment:
> also enables centralized management of all your secured clusters

Is there a clear way to distinguish possible senses of **centralized management**?
* If I understand correctly, **Cluster Status** of **all** makes sense, independent of installation method.
* However, management in an active sense is what these changes aim to limit to less than **all** clusters.
    Someone who knows more than me can comment if even **Delete cluster** makes sense for other installation methods than manifest.

Reply:
"Delete" is a little of a misnomer here, because it just removes the link to the cluster in the ACS database. I don't think it can actually remove the Secured Cluster components from the cluster that is being "deleted from our records". It most definitely does not **destroy** the cluster itself--that still exists, just without any monitoring by ACS.